app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
- [ ] [REQUIRED] I also noticed that you added explicit foreign keys (foreign_key: 'user_id' and foreign_key: 'post_id') to the belongs_to associations. According to the established design and ERD, these associations don't require explicit foreign keys.

For clarity and adherence to the intended structure, please update your associations